venvs/converge.py

```diff
@@ -92,6 +92,11 @@ def main(filesystem, locator, link_dir, handle_error):
 
         for link in config["link"]:
             name, _, to = link.partition(":")
+            for path in link_dir.heritage():
+                if not filesystem.exists(path):
+                    # TODO: add an `exists_ok` parameter
+                    # TODO: add a `create_parents` parameter
+                    filesystem.create_directory(path)
```

Comment:
This makes `main()` too complicated?  Fine...  guess I really have to implement it properly in `create_directory()`.  How dare a style check make me do the right thing.  `:|`

Reply:
Ha.